.eslintrc.js

```diff
@@ -114,6 +114,7 @@ module.exports = {
       "warn",
       { allowSameFolder: true, rootDir: "src" },
     ],
+    "jest/valid-title": ["error", { ignoreTypeOfDescribeName: true }],
```

Comment:
See [commit msg](https://github.com/artsy/force/pull/13041/commits/46f725beae43c08dd4159a533adada8c01b54c34) for the rationale. This would bite me later, so snuck this in as the first commit.

Without this change, ESLint was forcing me to change already existing lines like [this](https://github.com/artsy/force/blob/0336cdf1047b3495de79456e771357901e87942d/src/Components/ArtworkFilter/Utils/__tests__/urlBuilder.jest.tsx#L6), which I think are actually preferable in their current form, for reasons stated above^